firewall-examples.md

```diff
@@ -1,45 +1,33 @@
-Title: Cloud Firewall Examples
```

Comment:
This line and the "stability" line below are a special case. They're actually metadata that the publishing system uses. The syntax is `Fieldname: value` followed by a `---` on a line by itself.